packages/react/src/components/Toggle/index.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, boolean, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+
+import Toggle from "./";
+
+storiesOf("Buttons/Toggle", module)
```

Comment:
Should this be in the `Buttons` stories or the `Form controls` one? 🤔

Reply:
Dunno :/
No strong opinion. But in theory button would also be a part of form right?

Reply:
Dunno either, let's keep it there then. Or move it at the root, your call 🙂

Reply:
moved at root